syntax/vue.vim

```diff
@@ -40,13 +40,30 @@ function! s:register_language(language, tag, ...)
   endif
 endfunction
 
-if !exists("g:vue_disable_pre_processors") || !g:vue_disable_pre_processors
+function! s:register_tag(language, tag)
+  if s:syntax_available(a:language)
+    execute 'syntax include @' . a:language . ' syntax/' . a:language . '.vim'
+    unlet! b:current_syntax
+    execute 'syntax region vue_' . a:language
+          \ 'keepend'
+          \ 'start=/<' . a:tag . '>/'
```

Comment:
This lacks the `\_[^>]*` from `s:register_language()`, it's for maintaining the highlighting when the tag contains other, unrelated attributes.

However I think the `s:register_language()` function can be reused to highlight `<script>` and `<style>` as well, since it's not that different. I can have a further look at this too when I find some time